docs/grpc_tutorial.md

```diff
@@ -178,19 +178,19 @@ We expect that the gRPC learner part will often be on a compute cluster and be s
 To test the gRPC in a setup like this you can start the servers on the computer side and the client part separately.
 For one gRPC server:
 ```bash
-python3 ./grpc_examples/run_grpc_server.py --port 9995 --metrics_port 9091
+python3 ./colearn_grpc/scripts/run_grpc_server.py --port 9995 --metrics_port 9091
```

Comment:
This should use the -m version now

Reply:
I am not so sure about this one. The rest of the tutorial kind of assumes that the code is downloaded (since you need to modify it for your own factory). Thats why I left it as is.